### PR TITLE
Document that right now we rely on CMAKE_INSTALL_PREFIX=/usr

### PR DESCRIPTION
**Pull request by emilio**
We hard-code /usr/bin/mozillavpn in a couple places on Linux.

The right fix would be to make the relevant files:

 * `linux/extra/mozillavpn-startup.desktop`
 * `linux/extra/mozillavpn.desktop`
 * `linux/mozillavpn.service`
 * `src/platforms/linux/daemon/org.mozilla.vpn.dbus.service`

Generated at build-time, using the specified `CMAKE_INSTALL_PREFIX`
instead. But I'm not a CMake expert and I'm not sure what's the best way
to do that.

Instead, document that this flag is needed, since a lot of distros
default to /usr/local otherwise.

### CODE REVIEW

**Review by oskirby — APPROVED** [review]
I think this is fine, but it should no longer be necessary as soon as #3939 gets merged, and fixes our hard-coded dependency on `/usr/`

**Comment by emilio** [comment]
Ah, great, I was not aware of https://github.com/mozilla-mobile/mozilla-vpn-client/pull/3888, happy to close in favor of that (or if you think it's gonna take a while we should probably merge this in the meantime, your call)

**Pull request merged**
